### PR TITLE
Added ability to generate reports even if there are no tests in the project

### LINKED ISSUE #409: Add the ability to generate reports even if there are no tests in the project
From the point of view of using the plugin, it is illogical when reports stop being generated if all tests are excluded or tests have not yet been added to the project.

With missing tests, XML/HTML reports should be generated with zero coverage, and verification should be carried out with the assumption that all lines/methods/classes are not covered.

### PR DESCRIPTION
**Pull request by shanshin**
Resolves #409

### CODE REVIEW

File: kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/NoTestReportsTests.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    private fun checkReportsGenerated(build: GradleBuild) {
+        val buildResult = build.runWithParams("koverXmlReport", "koverHtmlReport", "koverVerify")
+        val checkerContext = build.createCheckerContext(buildResult)
+
+        checkerContext.xml(defaultXmlReport()) {
+            classCounter("kotlinx.kover.templates.ExampleClass").assertFullyMissed()
+        }
+        checkerContext.checkHtmlReport()
+        assertFalse(buildResult.isSuccessful)
+        assertContains(buildResult.output, "Rule violated: lines covered percentage is 0.000000, but expected minimum is 50")
```
Comment (sandwwraith) [inline kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/NoTestReportsTests.kt:34]: Should there be a test without tests and without rules — to check that report with 0 coverage generated, but the build is still succesfull?

File: kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/MultiProjectTests.kt [hunk 2]
```diff
@@ -113,16 +109,11 @@ internal class MultiProjectTests {
 
         run("koverXmlReport", "koverHtmlReport", "koverVerify") {
             checkDefaultBinReport(false)
-
-            checkOutcome("koverHtmlReport", "SKIPPED")
-            checkOutcome("koverXmlReport", "SKIPPED")
-            checkOutcome("koverVerify", "SKIPPED")
+            taskNotCalled(defaultTestTaskName(slice.type))
```
Comment (sandwwraith) [inline kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/MultiProjectTests.kt:112]: Shouldn't we do `checkOutcome("koverHtmlReport", "SUCCESS")`?

**Review by sandwwraith — APPROVED** [review]
